### PR TITLE
Bring Bazel to 2.2.0

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jdk:
 install:
   - export PATH=$PATH:$HOME/bin && mkdir -p $HOME/bin
   - wget https://github.com/bazelbuild/buildtools/releases/download/0.22.0/buildifier && chmod +x buildifier && mv buildifier $HOME/bin/
-  - wget https://github.com/bazelbuild/bazel/releases/download/1.0.0/bazel-1.0.0-linux-x86_64 && mv bazel-1.0.0-linux-x86_64 bazel && chmod +x bazel && mv bazel $HOME/bin/
+  - wget https://github.com/bazelbuild/bazel/releases/download/1.2.1/bazel-1.2.1-linux-x86_64 && mv bazel-1.2.1-linux-x86_64 bazel && chmod +x bazel && mv bazel $HOME/bin/
```
Comment (chanseokoh) [inline .travis.yml]: Hmm... this makes me question if this Bazel binary is ever used. I am pretty sure version 1.0.0 just won't work. Let me test something.

Comment (chanseokoh) [reply .travis.yml]: Huh, it does look like 1.0.0 has been working. Let's upgrade to 2.2.0 instead of 1.2.1. (Locally, I have been using 2.1.1.)

**Comment by chanseokoh** [comment]
It is still 1.2.1. Please push the right commit.

**Comment by gliptak** [comment]
@chanseokoh done

**Pull request merged**